registries/adapters/openshift_adapter.go

```diff
@@ -42,7 +42,12 @@ type OpenShiftImage struct {
 
 // RegistryName - Retrieve the registry name
 func (r OpenShiftAdapter) RegistryName() string {
-	return strings.TrimPrefix(r.Config.URL.String(), "https://")
+	name := strings.Split(r.Config.URL.String(), "://")
```

Comment:
Do we need to add the case conversion in here since the `url` ccannot cover the Uppercase? Like: `"http://FOO.REDHAT.COM"` should be chaned to `foo.redhat.com` after this `func`.

Reply:
No need to do this. The value of `r.Config.URL` is a `*url.URL`. So it would need to be a valid url BEFORE that was ever set. We go from string to URL early in the startup process. The parsing of the http://FOO.REDHAT.COM would fail before the broker starts up. So we're okay here.

Reply:
Thanks, I got it and also updated #85.